### PR TITLE
Adds cfn-bootstrap to getopt option string

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: AWSutils.sh [hunk 1]
```diff
@@ -360,8 +360,8 @@ function InstallCfnBootstrap {
 ## Main program-flow
 ######################
 OPTIONBUFR=$( getopt \
-   -o C:c:d:hi:m:s:t:\
-   --long cli-v1:,cli-v2:,help,instance-connect:,mountpoint:,ssm-agent:,systemd-services:,utils-dir: \
+   -o C:c:d:hi:m:n:s:t:\
+   --long cli-v1:,cli-v2:,cfn-bootstrap:,help,instance-connect:,mountpoint:,ssm-agent:,systemd-services:,utils-dir: \
```
Comment (ferricoxide) [inline AWSutils.sh:369]: Not alpha-sorted:

`cfn-bootstrap` is earlier than `cli-*`

Comment (lorengordon) [reply AWSutils.sh]: fixed

**Comment by lorengordon** [comment]
Build is succeeding: https://us-east-1.console.aws.amazon.com/codesuite/codebuild/701759196663/projects/spel-ci/build/spel-ci%3A84d4bd88-c0e7-40cf-8f33-362a8e6a983d?region=us-east-1

**Review by ferricoxide — APPROVED** [review]
I'm good, now

**Pull request merged**